tests/e2e/api/src/models/orders/shared/classes.ts

```diff
@@ -0,0 +1,412 @@
+import { MetaData } from '../../shared-types';
+import { Model } from '../../model';
+import { TaxStatus } from './types';
+
+/**
```

Comment:
Thinking out loud: taxes and the associated calculations (e.g., rounding) are important but error-prone. The model though is just for interacting with the API though, which does all the calculations, which takes off some of the pressure. Are we still confident that we're testing this part enough, or do we need some unit tests here? (Possibly I'll discover them further down the file list, not sure.)

Reply:
Adding additional API tests is on the post `API v0.2` list.